plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -34,9 +34,18 @@ var NodesTable = React.createClass({
     };
   },
 
+  renderRegion(_prop, node) {
+    const name = node.getRegionName();
+
+    return (
+      <span>
+        {name}
```

Comment:
This is missing the empty state `mdash`! Below, it is used in the sortFunction, but missing here. Should be `name || "\u2014"`

Reply:
Added this to line 38 but thanks that one slipped.